src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -471,11 +471,12 @@ public static IEnumerable<object[]> GetAsync_IPBasedUri_Success_MemberData()
             }
         }
 
-        [Fact]
-        public async Task GetAsync_IPv6AddressInHostHeader_CorrectlyFormatted()
+        [Theory]
+        [InlineData("http://", "[::1234]")]
+        [InlineData("http://", "[::1234]:8080")]
```

Comment:
Nit: the scheme doesn't need to be parameterized

Reply:
Will fix.